components/x-privacy-manager/src/messages.jsx

```diff
@@ -0,0 +1,78 @@
+import { h } from '@financial-times/x-engine';
+import s from './privacy-manager.scss';
+
+/**
+ * Provide a way to return to the referrer's homepage
+ * Potentially a Specialist Title, FT.com or FT App
+ *
+ * @param {string} referrer
+ */
+function renderReferrerLink(referrer) {
+	let url;
+
+	try {
+		url = new URL(`https://${referrer}`);
+	} catch (err) {
```

Comment:
Worth including the `err` parameter if unused?

Reply:
I'll use `_` instead 👍